src/payment_channel_management_strategies/DefaultPaymentChannelManagementStrategy.js

```diff
@@ -0,0 +1,61 @@
+import { find } from 'lodash';
+
+export default class DefaultPaymentChannelManagementStrategy {
```

Comment:
I would also add interface to implement custom strategy - not sure if interfaces are common in JS, thogh :-)

Reply:
Yeah, this would be communicated with api docs which would be made available soon.